picard/ui/metadatabox.py

```diff
@@ -186,6 +186,12 @@ def __init__(self, parent):
         self.changes_first_action.setChecked(config.persist["show_changes_first"])
         self.changes_first_action.toggled.connect(self.toggle_changes_first)
         self.browser_integration = BrowserIntegration()
+        # TR: Keyboard shortcut for "Add New Tag..."
+        QtGui.QShortcut(QtGui.QKeySequence(_("Alt+Shift+A")), self, partial(self.edit_tag, ""))
+        # TR: Keyboard shortcut for "Edit..." (tag)
+        QtGui.QShortcut(QtGui.QKeySequence(_("Alt+Shift+E")), self, partial(self.edit_selected_tag))
+        # TR: Keyboard shortcut for "Remove" (tag)
+        QtGui.QShortcut(QtGui.QKeySequence(_("Alt+Shift+R")), self, self.remove_selected_tags)
```

Comment:
Ok, better than nothing, in the future i hope we'll be able to use gettext contexts (see https://bugs.python.org/issue2504)